lib/FileApprover.js

```diff
@@ -28,8 +29,19 @@ exports.verify = function (namer, writer, reporterFactory, options) {
     writer.write(approvedFileName);
   }
 
+  var selectFirstCompatibleReporter = function () {
+    var reporter = _.find(reporterFactory(), function (reporterCandidate) {
```

Comment:
Could we consider using the native `filter` api? `reporterFactory().filter(...)` (Then we wouldn't have to depend on lodash here).

I know we use lodash elsewhere, would like to transition to more native api's where possible.